spec/new_spec.cr

```diff
@@ -1,7 +1,7 @@
 require "./spec_helper"
 
 describe Terminfo do
-  data = Terminfo::Data.new path: "/lib/terminfo/x/xterm"
+  data = Terminfo::Data.new path: "/lib/terminfo/x/xterm" # Not accurate on macOS, fails
```

Comment:
Let's modify the test to load the terminfo definition from a fixed/builtin copy that is distributed with Terminfo in ./filesystem/xterm. This would be preferred so that the test always loads known contents.

Reply:
@docelic I totally missed this directory. The test has been modified to leverage that file.

Reply:
Yes, this dir contains the files that get baked into the compiled binary and are available with `Terminfo.new builtin: ....`.

Reply:
@docelic As a note, the spec assumes for the final entry that `xterm` is the user's current terminal. I made some changes to that test to pick up on `TERM`, and to fall back to `xterm` only if it's not present. Does this seem reasonable?

Reply:
Yes, saw it, that was a nice catch. Thanks!